### PR TITLE
[docs] Correct examples to not use deprecated API

### PR DESCRIPTION
**Pull request by AndreyBelym**
\cc @AlexanderMoskovkin

### CODE REVIEW

File: docs/articles/documentation/test-api/actions/right-click.md [hunk 1]
```diff
@@ -21,14 +21,15 @@ The following example shows how to use the `t.rightClick` action to invoke a gri
...
+import { Selector } from 'testcafe';
 
 fixture `My fixture`
     .page `http://www.example.com/`;
 
 test('Popup Menu', async t => {
     await t.rightClick('#cell-1-1');
 
-    expect(await t.select('#cell-popup-menu')).to.be.null;
+    expect(await Selector('#cell-popup-menu')).to.be.null;
```
Comment (AlexanderMoskovkin) [inline docs/articles/documentation/test-api/actions/right-click.md]: This doesn't work. It should be `expect(await Selector('#cell-popup-menu')()).to.be.null;`

But I think our new built-in assertions will be awesome here:
```js
await t
    .rightClick('#cell-1-1')
    .expect(Selector('#cell-popup-menu').exists).eql(false);
```

Let's use it

File: docs/articles/documentation/test-api/actions/select-text.md [hunk 2]
```diff
@@ -120,8 +121,8 @@ test('Delete text within a contentEditable element', async t => {
         .selectEditableContent('#foreword', '#chapter-3')
         .pressKey('delete');
 
-    expect(await t.select('#chapter-2')).to.be.null;
-    expect(await t.select('#chapter-4')).to.not.be.null;
+    expect(await Selector('#chapter-2')).to.be.null;
```
Comment (AlexanderMoskovkin) [inline docs/articles/documentation/test-api/actions/select-text.md]: The same as above

File: docs/articles/documentation/test-api/actions/select-text.md [hunk 3]
```diff
@@ -110,18 +110,17 @@ This function works for HTML elements that have the `contentEditable` attribute
...
 
 test('Delete text within a contentEditable element', async t => {
     await t
         .selectEditableContent('#foreword', '#chapter-3')
-        .pressKey('delete');
-
-    expect(await t.select('#chapter-2')).to.be.null;
-    expect(await t.select('#chapter-4')).to.not.be.null;
+        .pressKey('delete')
+        .expect(Selector('#chapter-2').exists).eql(false)
```
Comment (helen-dikareva) [inline docs/articles/documentation/test-api/actions/select-text.md]: better use `ok` and `notOk`

Comment (AlexanderMoskovkin) [inline docs/articles/documentation/test-api/actions/select-text.md]: yep

File: examples/running-tests-using-travis-and-saucelabs/tests/index-test.js [hunk 4]
```diff
@@ -1,13 +1,10 @@
...
 
 test('My test', async t => {
     await t
-		.click('#click-here');
-
-    const button = await t.select('#click-here');
-
-    expect(button.value).to.equal('Hello!');
+		.click('#click-here')
+        .expect(Selector('#click-here').value).to.equal('Hello!');
```
Comment (AlexanderMoskovkin) [inline examples/running-tests-using-travis-and-saucelabs/tests/index-test.js]: just fix formatting (remove one space symbol before `.expect` and lgtm =)

File: docs/articles/documentation/test-api/assertions.md [hunk 5]
```diff
@@ -19,26 +19,21 @@ Examples in this documentation use [BDD-style Chai assertions](http://chaijs.com
...
 
 test('Test 1', async t => {
-    await t.click('#myelem');
-
-    const myElement = await t.select('#myelem');
-
-    expect(myElement.visible).to.be.true;
+    await t
+        .click('#myelem')
+        .expect(Selector('#myelem').visible).eql(true);
```
Comment (helen-dikareva) [inline docs/articles/documentation/test-api/assertions.md]: please use `ok()` too and it'll lgtm

File: examples/basic/test.js [hunk 6]
```diff
@@ -83,7 +83,7 @@ test('Moving the slider', async t => {
 
     await t
         .click(page.triedTestCafeCheckbox)
-        .dragToElement(page.slider.handle, page.slider.tick.with({ text: '9' }));
+        .dragToElement(page.slider.handle, page.slider.tick.withText('9'));
```
Comment (inikulin) [inline examples/basic/test.js]: Use our assertions in this example as well

Comment (inikulin) [inline examples/basic/test.js]: Also, I guess we should redesign this page model to be a class (so we'll be able to create selector derivatives). E.g. we can create single label selector: `const label = Selector('label')` and then expose other selectors as derivatives: `label: label.withText('Support for testing on remote devices')`. `interfaceSelectOption` can be a derivative of `interfaceSelect`: `interfaceSelect.find('option')`

File: examples/basic/test.js [hunk 7]
```diff
@@ -1,80 +1,76 @@
...
-
-    slider: {
-        handle: Selector('.ui-slider-handle'),
-        tick:   Selector('.slider-value')
-    },
-
-    interfaceSelect:       Selector('#preferred-interface'),
-    interfaceSelectOption: Selector('#preferred-interface > option')
-};
+class Page {
```
Comment (inikulin) [inline examples/basic/test.js]: Move `// Page model` comment above all page model-related stuff

File: examples/basic/test.js [hunk 8]
```diff
@@ -1,80 +1,76 @@
...
+        .typeText(page.nameInput, 'Peter Parker')           // Type name
+        .click(page.nameInput, { caretPos: 5 })             // Move caret position
+        .pressKey('backspace')                              // Erase a character
+        .expect(page.nameInput.value).eql('Pete Parker');   // Check result
 
-    await t.pressKey('home right . delete delete delete'); // Pick even shorter form for name
-
-    // Check result
-    expect(await page.nameInput.value).eql('P. Parker');
+    await t
```
Comment (inikulin) [inline examples/basic/test.js]: Can be combined into a single chain

**Comment by AndreyBelym** [comment]
Looks like AppVeyor is broken, I have created an issue: https://github.com/appveyor/ci/issues/1195

**Comment by AlexanderMoskovkin** [comment]
lgtm /cc @helen-dikareva

**Pull request merged**